components/Input/CustomInput.tsx

```diff
@@ -29,10 +33,12 @@ const CustomInput: React.FC<CustomInputProps> = ({
 	return (
 		<CustomInputContainer style={style} className={className}>
 			<input
+				id={id}
+				defaultValue={defaultValue}
 				disabled={disabled}
 				placeholder={placeholder}
 				value={value}
-				type="number"
+				inputMode={id ? 'decimal' : undefined}
```

Comment:
This change breaks key functionality on the futures page. We are no longer preventing input of non-numeric characters in position sizes:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/10401554/164279204-d357eb6d-a755-4a27-b141-da043985443a.png">

This also throws an error when you input some non-numeric character.